skll/experiments.py

```diff
@@ -282,20 +282,28 @@ def _print_fancy_output(learner_result_dicts, output_file=sys.stdout):
     print('Feature Set: {}'.format(lrd['featureset']), file=output_file)
     print('Learner: {}'.format(lrd['learner_name']), file=output_file)
     print('Task: {}'.format(lrd['task']), file=output_file)
+    if lrd['folds_file']:
+        print('Specified Folds File: {}'.format(lrd['folds_file']),
+              file=output_file)
     if lrd['task'] == 'cross_validate':
         print('Number of Folds: {}'.format(lrd['cv_folds']),
               file=output_file)
-        print('Stratified Folds: {}'.format(lrd['stratified_folds']),
-              file=output_file)
+        if not lrd['cv_folds'].endswith('folds file'):
```

Comment:
Is "folds file" supposed to be "folds_file" here? Can we just make it check if the string is equal to "folds file"/"folds_file" rather than using endswith or is there some reason that might not work?

Reply:
Nope, it's trying to match the string on the right hand side of the colon in the results file which should say `3 via folds file` so it will always ends with "folds file" not "folds_file".

Reply:
Ok, I see. Misread that.

Reply:
No worries. If you want to see some example output for various conditions, you can play around with `cross_validate.cfg` in the titanic example and look at the results files.